R/github-pages.R

```diff
@@ -0,0 +1,164 @@
+#' Configure a GitHub Pages site
+#'
+#' Activates or reconfigures a GitHub Pages site for a project hosted on GitHub.
+#' This function anticipates two specific usage modes:
+#' * Publish from the root directory of a `gh-pages` branch, which is assumed to
+#'   be only (or at least primarily) a remote branch.
+#'   - Typically the `gh-pages` branch is managed by an automatic "build and
```

Comment:
I don't love this nested list style. I think you could just run them into the top-level list.